src/DDTrace/Integrations/Slim/SlimSandboxedIntegration.php

```diff
@@ -43,7 +43,7 @@ public function init()
             $rootSpan->setTag(Tag::SERVICE_NAME, $appName);
 
             // Hook into the router to extract the proper route name
-            \dd_trace_method('Slim\Router', 'lookupRoute', function (SpanData $span, $args, $return) use ($rootSpan) {
+            \DDTrace\trace_method('Slim\Router', 'lookupRoute', function (SpanData $s, $a, $return) use ($rootSpan) {
```

Comment:
Just asking to adopt the same approach in similar scenarios. Do we want to use `$s` and `$a` meaning 'not used'? Something like the `_` of other languages?

Reply:
I like this idea! I changed these just to get the linter to pass, but having a consistent "not used" convention would be good to implement across the board. Are we good with the first initial of each param: `$s`, `$a`, `$r`? Or something else?